samples/GuestConfiguration/audit-installed-application-linux/azurepolicy.json

```diff
@@ -0,0 +1,32 @@
+{
+    "properties": {
+        "displayName": "Audit that an application is installed inside Linux VMs",
+        "policyType": "BuiltIn",
```

Comment:
remove policyType: BuiltIn here as it would prevent users from deploying the sample

Reply:
fixed